java/src/main/java/com/saucelabs/saucebindings/SauceOptions.java

```diff
@@ -326,7 +326,38 @@ private void setEnumCapability(String key, String value) {
         }
     }
 
-    protected String getEnvironmentVariable(String key) {
+    private MutableCapabilities addAuthentication() {
+        MutableCapabilities caps = new MutableCapabilities();
+        caps.setCapability("username", getSauceUsername());
+        caps.setCapability("accessKey", getSauceAccessKey());
+        return caps;
+    }
+
+    String getSauceUsername() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=5305593)